k8s/memcached/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,55 @@
 actions:
-- name: dummy test
+- name: StatefulSet nodes deployed successfully
```

Comment:
You will be able to safely remove it due to a new release of k8s tools in version 0.7.5 that has improved application readiness in tests.

https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/releases/tag/0.7.5

Reply:
Done. Thanks for implementing that feature!